src/ucp/api/ucp_def.h

```diff
@@ -321,11 +321,11 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 
  /**
  * @ingroup UCP_COMM
- * @brief Completion callback for non-blocking sends ucp_tag_send_nbx call.
+ * @brief Completion callback for non-blocking sends.
  *
- * This callback routine is invoked whenever the @ref ucp_tag_send_nbx
- * "send operation" is completed. It is important to note that the call-back is
- * only invoked in a case when the operation cannot be completed in place.
+ * This callback routine is invoked whenever the @ref ucp_tag_send_nbx,
+ * @ref ucp_am_send_nbx, ucp_stream_send_nbx, ucp_put_nbx, ucp_get_nbx,
```

Comment:
@ref missing before few function names

Reply:
Please generate doxygen before you merge to make sure that it creates ref correctly.

Reply:
![image](https://user-images.githubusercontent.com/14221180/144885119-52599f3a-5e9e-40b8-a215-f441c7d682b3.png)

@shamisp, verified

Reply:
Awesome - thanks for doing this !